cicecore/cicedynB/general/ice_flux.F90

```diff
@@ -158,9 +160,9 @@ module ice_flux
       real (kind=dbl_kind), dimension (:,:,:), allocatable, public :: &
          zlvl    , & ! atm level height (momentum) (m)
          zlvs    , & ! atm level height (scalar quantities) (m)
-         uatm    , & ! wind velocity components (m/s)
-         vatm    , &
-         wind    , & ! wind speed (m/s)
+         uatm    , & ! wind velocity components (m/s), on grid_atm_dynu
+         vatm    , & !                                 on grid_atm_dynv
+         wind    , & ! wind speed (m/s)              , on grid_atm_dynu
```

Comment:
Are there explicit shifts of variables needed for thermo calculations (like wind) from grid_atm_dyn* to the T grid, for Icepack?

Reply:
As currently implemented, uatm and vatm are mapped to the "T" grid in ice_step_mod.F90 and passed into icepack where strair is computed on the T grid.  strair is then mapped to the U, N, or E grids in the dynamics as needed.  So I think this is fine.  In the future, it could be that we compute strair on the U, N, or E grid directly and then we'd need to modify the implementation.

One of the general challenges in supporting flexible grid coupling/forcing is to keep track of how the different forcing/coupling variables are used and to keep the implementation up to date.